lib/bootstrap_form/form_builder.rb

```diff
@@ -21,9 +21,9 @@ def initialize(object_name, object, template, options)
       @control_col = options[:control_col] || default_control_col
       @label_errors = options[:label_errors] || false
       @inline_errors = if options[:inline_errors].nil?
-        @label_errors != true
+        !@label_errors
```

Comment:
`@label_errors` isn't necessarily a boolean, so I think the comparison to a boolean is valid here.